Packs/Core/Integrations/CoreIOCs/CoreIOCs.yml

```
@@ -0,0 +1,141 @@
+    Cortex XDR.
+  defaultvalue: reputation:Bad and (type:File or type:Domain or type:IP)
+  display: Sync Query
+  hidden: false
+  name: query
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Use the Cortex XDR - IOCs integration to sync indicators from Cortex
+  XSOAR to Cortex XDR and back to Cortex XSOAR. Cortex XDR is the world's first detection
+  and response app that natively integrates network, endpoint and cloud data to stop
+  sophisticated attacks.
+display: Indicators detection
+name: Cortex Core - IOC
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      default: true
+      defaultValue: 'false'
+      description: |-
+        For first sync, set to true.
+        (do NOT run this twice!)
+      isArray: false
+      name: firstTime
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    deprecated: false
+    description: Sync your IOC with Cortex XDR and delete the old.
+    execution: false
+    name: core-iocs-sync
+  - arguments:
+    - default: false
+      description: IOCs to push. leave empty to push all recently modified IOCs.the
+        indicators
+      isArray: true
+      name: indicator
+      required: false
+      secret: false
+    deprecated: false
+    description: Push modified IOCs to Cortex XDR.
+    execution: false
+    name: core-iocs-push
+  - arguments:
+    - default: false
+      description: The time of the file creation (use UTC time zone).
+      isArray: false
+      name: time
+      required: true
+      secret: false
+    deprecated: false
+    description: Set sync time manually (Do not use this command unless you unredstandard
+      the consequences).
+    execution: false
+    name: core-iocs-set-sync-time
+  - deprecated: false
+    description: Creates the sync file for the manual process. Run this command when
+      instructed by the XDR support team.
+    execution: false
+    name: core-iocs-create-sync-file
+  - arguments:
+    - default: false
+      description: The indicator to enable.
+      isArray: true
+      name: indicator
+      required: true
+      secret: false
+    deprecated: false
+    description: Enables IOCs in the XDR server.
+    execution: false
+    name: core-iocs-enable
+  - arguments:
+    - default: false
+      description: The indicator to disable.
+      isArray: true
+      name: indicator
+      required: true
+      secret: false
+    deprecated: false
+    description: Disables IOCs in the XDR server.
+    execution: false
+    name: core-iocs-disable
+  dockerimage: demisto/python3:3.10.1.25933
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+- No tests
+fromversion: 5.5.0
```

```suggestion
fromversion: 6.2.0
```
maybe should be even higher. need to check